src/lib/core/mp_util.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
Please factor out addition of `mp_util.h` into a separate patch and update `box.cc` to use `mp_format_on_region` in that patch.

Rationale: We may use these methods in a patch that needs to be backported to 2.10. If this happen, it'll be much easier to backport these methods to 2.10 if there's a separate patch introducing them.

Reply:
Done.